momepy/functional/_diversity.py

```diff
@@ -2,11 +2,14 @@
 
 import numpy as np
 import pandas as pd
+import scipy as sp
 from libpysal.graph import Graph
 from numpy.typing import NDArray
 from packaging.version import Version
 from pandas import DataFrame, Series
 
+import momepy as mm
```

Comment:
I would prefer to load just the functions you need. Any maybe check if we can njit them :)